metadata-ingestion/src/datahub/ingestion/source/sql/mssql/job_models.py

```diff
@@ -267,3 +294,15 @@ def as_maybe_platform_instance_aspect(self) -> Optional[DataPlatformInstanceClas
                 ),
             )
         return None
+
+    @property
+    def as_container_aspect(self) -> ContainerClass:
+        databaseKey = DatabaseKey(
+            platform=self.entity.orchestrator,
+            instance=self.entity.platform_instance
+            if self.entity.platform_instance
+            else None,
```

Comment:
the ternary doesn't do anything here
```suggestion
            instance=self.entity.platform_instance,
```

Reply:
pushing...